Sources/App/Logger/JSONLogger.swift

```diff
@@ -52,6 +52,7 @@ extension JSONLogger {
             let df = DateFormatter()
             df.dateFormat = "yyyy-MM-dd'T'hh:mm:ss.SSSZ"
             df.locale = Locale(identifier: "en_US_POSIX")
+            df.timeZone = TimeZone(secondsFromGMT: 0)
```

Comment:
That's what happens when you implement and test something while visiting London last week: now that I'm back in Paris the test failed locally because of me being back in GMT+1 😛